app/browser/actions/tabActions.js

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const dispatchAction = require('../../common/dispatcher/dispatchAction')
+
+const tabActions = {
+  didFinishNavigation: function tabActionsDidFinishNavigation (tabId, navigationState, windowId) {
```

Comment:
this seems clunky to me, anyone have a good idea for converting method names to strings that doesn't involve a constants file?